src/components/common/layout/languageToggle.njk

```diff
@@ -0,0 +1,18 @@
+<nav class="language-select" aria-label="Choose a language">
+    <ul class="govuk-list">
+        {% for lang in ['en', 'cy'] %}
+            <li class="govuk-header__navigation-item">
```

Comment:
I'm interested in the reasoning behind the choice to use the `govuk-header__navigation-item` class. 

Seeing as this class is actually specific to the navigational links in the Design System [header component](https://design-system.service.gov.uk/components/header/), which is unrelated to this component, I would suggest that relying on this class could lead to unintended consequences in the future (it has actually already led to unintended consequences – you have had to over-write some of the styles that this class comes with 😄). 
It might be best to remove this class from the `li`, opting to use a BEM style class such as `language-select__list-item` instead.

Reply:
It was a left over from hmrc component I forgot to replace. They're using in it in a different layout so for them makes sense but not for us. I have updated it accordingly.